Frontend/src/components/Marker/marker.tsx

```diff
@@ -0,0 +1,50 @@
+import { LatLng, Marker as MarkerType } from "leaflet";
+import { useMemo, useRef, useState, type FC } from "react";
+import { Marker, Popup } from "react-leaflet";
+import "leaflet.utm";
+import "./marker.css";
+
+interface CustomMarkerProps {
+  title: string;
+  initialPosition: LatLng;
+}
+
+export const CustomMarker: FC<CustomMarkerProps> = ({
```

Comment:
The exported component should be named the same as the file it is in (Either change the component's name or the file name)